meta2v2/meta2_server.c

```diff
@@ -149,21 +149,11 @@ sqlx_service_resolve_peers(struct sqlx_service_s *ss,
 	return err;
 }
 
-#if 0
-static GError *
-sqlx_service_reply_no_peers(struct sqlx_service_s *ss UNUSED,
-		const struct sqlx_name_s *n UNUSED, gboolean nocache UNUSED,
-		gchar ***result UNUSED)
```

Comment:
This one is actually referenced in a comment a few lines below. You may want to remove the comment (or keep this function).